sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultServiceSpec.java

```diff
@@ -394,6 +409,16 @@ public Builder replacementFailurePolicy(ReplacementFailurePolicy replacementFail
         }
 
         /**
+         * Sets the policy as to whether to opt int to using GPU_RESOURCES offered by Mesos.
+         * @param optInToGpuResources opt in, boolean
+         * @return a reference to this Builder
+         */
+        public Builder gpuResourcePolicy(Boolean optInToGpuResources) {
```

Comment:
Last time I'll bring it up.  GPUs are resources and should be specified in `ResourceSet`s